tools/cloud_functions/gcs_event_based_ingest/tests/gcs_ocn_bq_ingest/test_monitoring_tables.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# $1 is a query string to dry_run
+function dry_run_query() {
+  bq query \
+    --use_legacy_sql=false \
+    --dry_run \
+    "$1"
+}
+
+echo "setting DATASET env variable"
+export DATASET='test_monitoring_dataset'
```

Comment:
there may be parallel tests running.
dataset id should be unique.
I suggest suffix with BUILD_ID which is [set by GCB automatically](https://cloud.google.com/cloud-build/docs/configuring-builds/substitute-variable-values).
```suggestion
export DATASET='test_monitoring_dataset'
```
```suggestion
export DATASET="test_monitoring_dataset_${BUILD_ID}"
```